src/js/vic-v2/config/form.js

```diff
@@ -202,6 +204,9 @@ const formConfig = {
               photo: {
                 type: 'object',
                 properties: {
+                  'view:dataURL': {
```

Comment:
This seems like a reasonable way to store the image data (to be stored on photo submission, and retrieved in `fetchInProgressForm`) that we need for the review page, but other ideas for handling this are welcome.

Reply:
This is what @rlbaker is working on now. We can't store this data url in the database without performance issues, so he is working on getting us a way to request the image based on the guid associated with the image. For now, what I did in the screen reader PR is save the file object itself in the form data. The actual file won't be included when it's serialized to be saved in progress, so it doesn't cause performance issues, and we'll be able to show previews using it until Ryan's work is done.

Reply:
Thanks for the context. I'll follow your lead and save the file object itself as well.